migration/migration_test.go

```diff
@@ -0,0 +1,71 @@
+package migration
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/almighty/almighty-core/configuration"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+)
+
+func TestConcurrentMigrations(t *testing.T) {
+	resource.Require(t, resource.Database)
+
+	var db1 *gorm.DB
+
+	var err error
+	if err = configuration.Setup(""); err != nil {
+		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+	}
+
+	db1, err = gorm.Open("postgres",
+		fmt.Sprintf("host=%s port=%d user=%s password=%s sslmode=%s",
```

Comment:
Can you please make sure to specify the database like [here](https://github.com/kwk/almighty-core/blob/d1d9a360146fdc5e152748b6b4f8802dab56f0b9/main.go#L79) as well?